### DISCUSSION TITLE
.tech domain in GITHUB student developer pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65215] [GitHub Education]
### Hi everyone,

why .tech domain not available now in github student developer pack for registration 1 year free?

**Comment by arafajs** [comment]
The offer was available until February 2023, but it has since been discontinued.

**Reply by ghost** [reply]
That means, If anyone applied for student developer pack in august 2023,  and their student developer pack also approved by github, then they cannot get .tech domain for 1 year registration on that pack ?

**Reply by arafajs** [reply]
@gopalstudent 
That's correct. If you applied for the GitHub Student Developer Pack in August 2023, and your pack was approved, you will not be able to get a .TECH domain for one year free as part of the pack. The offer for a free .TECH domain ended in February 2023.

**Reply by ghost** [reply]
Is there any other methodology or way to register and get .tech domain for free 1 year again? @arafajs

**Reply by arafajs** [reply]
Unfortunately no.
